app/navigation/OfficeSuppliesNavigator.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import controllers.journeys.expenses.officeSupplies.routes.{OfficeSuppliesCYAController, OfficeSuppliesDisallowableAmountController}
+import controllers.standard.routes.JourneyRecoveryController
+import models.database.UserAnswers
+import models.{CheckMode, Mode, NormalMode}
+import pages.Page
+import pages.expenses.officeSupplies.{OfficeSuppliesAmountPage, OfficeSuppliesDisallowableAmountPage}
+import play.api.mvc.Call
```

Comment:
did we decide we were going to have every subjourney in it's own navigator or that we'd put all non-tailoring expenses in the expensesNavigator?

Reply:
No, we didn't decide to do it this way. I just had to do something then and there. What do you think is best?

Reply:
I think if we have all the expenses in one navigator, the file will be quite large. Maybe we should chat to @lukastymo - what do you think?